### DISCUSSION TITLE
copy/paste not working via unraid webUI

### DISCUSSION THREAD

**Discussion by wally-nl** [jlesage/docker-cloudberry-backup#34] [Q&A]
Not sure if it's even possible but I just deployed this cloudberrybackup container on my freshly built unraid nas and have a nightmare getting my license activated as copy/paste using the browser doesn't seem to work. Not sure if that is something that can be easily fixed/worked around as it looks like a remote desktop/xwindows session in my browser.

**Comment by jlesage** [comment]
Copy/paste should work when using the clipboard from the left pop-up menu.
